src/main/java/spaceEngineers/environments/SocketEnvironment.java

```diff
@@ -32,6 +33,8 @@
     SocketEnvironment(String host, int port) {
 
         int maxWaitTime = 20000;
+        int socketConnectionTimeout = 4000;
+        int socketDataTimeout = 4000;
```

Comment:
It's not critical, but I like when physical quantities have units in the name of the variable, such as:

maxWaitTimeMs
socketConnectionTimeoutMs
etc.

Reply:
Just a Java newbie question: why not add something like final / const?

Reply:
I renamed variables to match this.

Yes good point, final should be in 99% places of the java code, including method parameters. There's even settings in IDEA (or Eclipse), that adds it all automatically on save. I used to do that before switching to kotlin.
These variables don't make sense at all and they should be (final) constructor parameters with some sane defaults. imho :-)
In kotlin, 99% variables should be "val", which serves the same purpose.